conf/pulsar_tools_env.sh

```diff
@@ -48,7 +48,7 @@ PULSAR_MEM="-Xmx128m -XX:MaxDirectMemorySize=128m"
 PULSAR_GC=" -client "
 
 # Extra options to be passed to the jvm
-PULSAR_EXTRA_OPTS="${PULSAR_EXTRA_OPTS} ${PULSAR_MEM} ${PULSAR_GC} ${PULSAR_GC_LOG} -Dio.netty.leakDetectionLevel=disabled"
+PULSAR_EXTRA_OPTS="${PULSAR_MEM} ${PULSAR_EXTRA_OPTS} ${PULSAR_GC} ${PULSAR_GC_LOG} -Dio.netty.leakDetectionLevel=disabled"
```

Comment:
+1. One slight improvement:
Please move `${PULSAR_EXTRA_OPTS}` as the very last entry so that it will also be possible to override `io.netty.leakDetectionLevel` when needed. 

btw. I happened to do a similar change in `bin/pulsar` script recently in https://github.com/apache/pulsar/pull/13025/files#diff-89618c73d511b2015766c1fb0cd4bd16aa7f949dfdfa02d1995b586b8ed57d7aL283

Reply:
Thank you, done